src/dlgProfilePreferences.cpp

```diff
@@ -2119,15 +2121,29 @@ void dlgProfilePreferences::saveMap()
         return;
     }
 
-    QString fileName =
-            QFileDialog::getSaveFileName(this, tr("Save Mudlet map"), mapSaveLoadDirectory(pHost), tr("Mudlet map (*.dat *.json);;", "Do not change the extension text (in braces) - it is needed programmatically!"));
-    if (fileName.isEmpty()) {
-        return;
-    }
+    auto datFilter = tr("Mudlet binary (*.dat)", "Do not change extensions (in braces) as they are used programmatically");
+    auto jsonFilter = tr("Mudlet JSON map (*.json)", "Do not change extensions (in braces) as they are used programmatically");
+    auto saveExtensions(QStringList() << datFilter << jsonFilter);
+
+    QFileDialog dialog(this);
+    dialog.setWindowTitle(tr("Save Mudlet map"));
+    dialog.setDirectory(mapSaveLoadDirectory(pHost));
+    dialog.setNameFilter(saveExtensions.join(";;"));
```

Comment:
Don't use raw string literals as that can leave it ambiguous to others whether the text should be translated or (in this case) not:
```suggestion
    dialog.setNameFilter(saveExtensions.join(QStringLiteral(";;")));
```